src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -44,6 +44,7 @@
 
             <button type="button"
                     class="btn btn-flat filter-button"
+                    [attr.aria-label]="filter.length === 0 ? 'Search' : 'Close'"
```

Comment:
Unfortunately we can't hard code aria-label values because these are human facing and need to be localizable. (The time picker has hard coded aria-labels but this is a pre-existing fault.)